src/index.js

```diff
@@ -5,7 +5,7 @@ import moment, { relativeTimeRounding } from 'moment'
 import { relative } from 'path';
 import { io } from 'socket.io-client';
 
-let tableData = {}, filterUsername = "", startDate = new Date(0), endDate = new Date(), localDate = new Date();
+let tableData = {}, filterUsername = "", startDate = new Date(2020, 10, 21, 0, 0, 0), endDate = new Date(), localDate = new Date();
```

Comment:
@RonLek I think we must keep it flexible instead of hard coding the start date(since it may be misleading for someone who doesn't know that the start date is set to 21-10-2020 by default), if the user needs to filter the PRs from the start date 21-10-2020, then one can simply select the custom date filter. 

@Sing-Li what do you think?

Reply:
I'd suggest to follow @djcruz93 's suggestion. But we need to provide cache mechanism using localStorage so that once selected custom date filter the next time opening gsoc.rocket.chat, we need to apply the filters for them by default. In this way, users don't need to select filters again.

Reply:
Let's move on this improvement into a new PR.